docs/ui-coverage/configuration/elementfilters.mdx

```diff
@@ -8,15 +8,21 @@ sidebar_label: elementFilters
 
 <UICovAddon />
 
+By default, every interactive and visible element is included in UI Coverage. The `elementFilters` property allows you to specify selectors for elements that should be excluded from these reports. This configuration is particularly useful for removing irrelevant or non-actionable elements to streamline coverage analysis.
+
+## Why use element filters?
+
 :::info
-Note: setting `elementFilters` impacts both Accessibility and UI Coverage
+**Note:** setting `elementFilters` impacts both Accessibility and UI Coverage
 reports. Nesting this property under an `accessibility` or `uiCoverage` key is
 supported, if you need to split them up.
 :::
 
-By default, every interactive and visible element is included in UI Coverage. The `elementFilters` property allows you to specify selectors for elements that should be excluded from UI Coverage.
+### Common use cases
 
-For every element considered interactive and visible by UI Coverage, the first `elementFilters` rule for which the `selector` property matches the element is used to either include or exclude the element based on the `include` value. Elements that do not match any rules are included by default.
+- **Excluding Irrelevant Elements**: Decorative icons, marketing banners, or tooltips that do not require testing can clutter reports. Use elementFilters to exclude them.
+- **Streamlining Reports**: Reducing noise by filtering out non-essential elements makes coverage reports more actionable.
+- **Improving Grouping Accuracy**: Avoid incorrectly grouping elements that share attributes but serve different purposes.
```

Comment:
not necessarily. another use case is ignoring third party components that you don't need to test

Reply:
as in "Avoid incorrectly grouping elements that share attributes but serve different purposes" does not sound like something elementFilters helps with